### DISCUSSION TITLE
Connections Going on top of the Nodes instead of behind

### DISCUSSION THREAD

**Discussion by keerthankrishna** [joyceworks/flowchart-react#24] [Q&A] [answered]
Hello there - not sure if this is the right place to ask this question but how do you manage having the connections go behind the nodes rather than on them ? I believe there's no Y axis right to set what is in front and what is behind ?

See below example - the red line is going on top of my box when i have it connected to another node above from somewhere below:
![image](https://user-images.githubusercontent.com/24373382/201790574-10bdd850-c6b5-4cce-b433-8462d61b413e.png)

**Comment by iamppz** [comment]
Hello, thank you for the feedback, I will work on it later.

**Reply by iamppz** [reply]
Please upgrade flowchart-react to 3.7.0 and use `connectionPosition="bottom"` to change connection position.

**Reply by keerthankrishna** [reply]
I tried this but I still see the same problem - thoughts ? Not sure if I'm putting something wrong here.
I tried both **bottom** and **top** but it gives the same output where it is going over the _Start_ node.
![image](https://user-images.githubusercontent.com/24373382/202557911-d60a819c-8e00-4e74-ae90-c258867dc0ad.png)

**Reply by iamppz** [reply]
Property `id` is deleted.

**Comment by iamppz** [comment, marked as answer]
<flowchart connectionPosition=bottom …/>

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Keerthan ***@***.***>
发送时间: Friday, November 18, 2022 4:55:46 AM
收件人: joyceworks/flowchart-react ***@***.***>
抄送: iamppz ***@***.***>; Comment ***@***.***>
主题: Re: [joyceworks/flowchart-react] Connections Going on top of the Nodes instead of behind (Discussion #24)


I tried this but I still see the same problem - thoughts ? Not sure if I'm putting something wrong here.
[image]<https://user-images.githubusercontent.com/24373382/202557911-d60a819c-8e00-4e74-ae90-c258867dc0ad.png>

―

**Reply by keerthankrishna** [reply]
Perfect, thank you so much! That worked.
